[docs] [2.20.1, 2.18.5] added editable fields for multiple cloud providers (#20626)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md

To view a provider, select it in the list of AWS Configs to display the **Overview**.

> To edit the provider, select **Config Details**, make changes, and click **Apply Changes**. For more information, refer to [Provider settings](#provider-settings). Note that for YBA version 2.20.1 and later, depending on whether the provider has been used to create a universe, you can only edit a subset of fields such as the following:

Reviewer: ```suggestion
To edit the provider, select **Config Details**, make changes, and click **Apply Changes**. For more information, refer to [Provider settings](#provider-settings). Note that for YBA version 2.20.1 and later, if the provider has been used to create a universe, you can only edit a subset of fields, including the following:
```

- Provider Name
- Access Key ID
- Secret Access Key
- Regions - You can add regions and zones to an in-use provider.

> Note that you cannot edit existing region details (such as shared subnet, yb image, and so on), or delete a region if any of the region's zones are in use. You also cannot delete zones which are in-use.

Author: After more clarification with @Jethro-M , I agree that we don't need to call out the examples at all as the fields are anyway not editable.
